robottelo/config/__init__.py

```diff
@@ -1,15 +1,19 @@
-import logging
+from pathlib import Path
 
 from dynaconf import LazySettings
 from dynaconf.validator import ValidationError
 
 from .validators import validators as dynaconf_validators
-from robottelo.config.base import ImproperlyConfigured
 from robottelo.config.base import Settings as LegacySettings
 from robottelo.config.facade import SettingsFacade
 from robottelo.config.facade import SettingsNodeWrapper
+from robottelo.errors import ImproperlyConfigured
+from robottelo.logging import config_logger as logger
 
-logger = logging.getLogger('robottelo.config')
+
+# Path instance for the Robottelo project root directory
+robottelo_root_dir = Path().resolve().parent.parent
+robottelo_log_dir = robottelo_root_dir.joinpath('logs')
```

Comment:
+100!

Reply:
i see these are also defined in robottelo.logging. could you use the same values defined there in place of these as well?

Reply:
Because `robottelo.config.__init__` and `robottelo.config.facade` and `robottelo.logging` import from each other, trying to import them turns on 'circular import mode' where everything blows up.

Maybe there is a better third place for these, or we can consolidate them when the facade gets removed and imports aren't quite so circular.

Reply:
I don't see any robottelo imports in robottelo.logging, so that might be the best place for the calculation and config could take the values from there. Ultimately, the computational impact is low, so this isn't close to a blocker.